ui/src/app/shared/components/webcomponents/upload-entities-modal.component.ts

```diff
@@ -10,17 +12,41 @@ import {TestDevice} from "../../../models/test-device.model";
 export class UploadEntitiesModalComponent extends LinkedEntitiesModalComponent {
 
   constructor(
+    public testPlanService : TestPlanService,
     @Inject(MAT_DIALOG_DATA) public modalData: any) {
     super(modalData);
   }
 
+
   ngOnInit(): void {
+    this.setTestPlans(this.modalData?.linkedEntityList.cachedItems)
   }
 
+  setTestPlans(testdevices){
+   let testPlanIds = [];
+   testdevices.forEach((data: TestDevice) => {
+     testPlanIds.push(data.testPlanId);
+   })
+    this.testPlanService.findAll("id@"+ testPlanIds.join("#")).subscribe(
+      res=> {
+        this.setTestPlanNames(res.content)
+      },
+      error=>{console.log(error)}
+      )
+      }
```

Comment:
@Raksha-testsigma Just correct the indentation of the brackets

Reply:
Done